test/appium/support/testrail_report.py

```diff
@@ -102,3 +101,10 @@ def add_results(self):
                     'comment': '%s' % ('# Error: \n %s \n' % emoji.demojize(last_testrun.error)) + devices + test_steps if last_testrun.error
                     else devices + test_steps}
             self.post(method, data=data)
+
+    def get_test_result_link(self, test_run_id, test_case_id):
+        try:
+            test_id = self.get('get_results_for_case/%s/%s' % (test_run_id, test_case_id))[0]['test_id']
+        except KeyError:
+            test_id = None
+        return '%stests/view/%s' % (self.url, test_id)
```

Comment:
is it okay to get tests/view/None in the end?

Reply:
updated